setup.py

```diff
@@ -34,6 +34,7 @@
     # Author details
     author="Adafruit Industries",
     author_email="circuitpython@adafruit.com",
+    python_requires=">=3.7.0",
     install_requires=["Adafruit-Blinka>=6.20.4", "adafruit-circuitpython-typing"],
```

Comment:
Since 7.0.0 of blinka is now out:
```suggestion
    install_requires=["Adafruit-Blinka>=7.0.0", "adafruit-circuitpython-typing"],
```